src/Employer/Employer.Web/Views/CreateVacancyOptions/Options.cshtml

```diff
@@ -1,7 +1,9 @@
 ﻿@model Esfa.Recruit.Employer.Web.ViewModels.CreateVacancyOptions.CreateVacancyOptionsViewModel
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        
+        <esfaFeatureDisabled name="@FeatureNames.AllowLevyPayingEmployersOnly">
+            <a asp-show="@Model.HasClonableVacancies" asp-route="@RouteNames.Dashboard_Index_Get" class="govuk-back-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">Return to your vacancies</a>
```

Comment:
Shouldn't this be going to `RouteNames.Dashboard_Account_Home`

Reply:
Return to your vacancies, in the prototype, takes flow back to the employer dashboard page.